resources/Orders ingestion to IBM Supply Chain Intelligence Suite from Microsoft Dynamics 365.yaml

```diff
@@ -0,0 +1,392 @@
+$integration: http://ibm.com/appconnect/integration/v2/integrationFile
+integration:
+  type: trigger-action
+  trigger-interfaces:
+    trigger-interface-1:
+      type: event-trigger
+      triggers:
+        SCHEDULE:
+          input-context:
+            data: scheduler
+          assembly:
+            $ref: '#/integration/assemblies/assembly-1'
+          options:
+            subscription:
+              scheduleConfiguration:
+                interval:
+                  unit: hour
+                  value: 1
+                  runOnceOncheck: true
+                  days:
+                    - MON
+                    - TUE
+                    - WED
+                    - THU
+                    - FRI
+                    - SAT
+                    - SUN
+                  timeZone: UTC
+      connector-type: streaming-connector-scheduler
+      account-name: Account 3
```

Comment:
Should remove account definitions, as per guidance:
"Important: Make sure that you delete all account-name elements from the file either before or after you upload the file. (That will then force users to connect to their own accounts.)"
See https://github.ibm.com/Cloud-Integration/appconnect-wiki/wiki/Authoring-App-Connect-templates